spring-security-web-app/README.md

```diff
@@ -0,0 +1,225 @@
+---
+page_type: sample
+languages:
+- java
+- html
+author: ramya25
+products:
+- spring security
+- azure-active-directory
+description: "This sample demonstrates a Java web application showcasing how to use Spring security to validate the access token obtained using OAuth2.0."
+urlFragment: ms-identity-java-webapp
+---
+
+# A Java web application using Spring security which signs in users with the Microsoft identity platform
+
+## About this sample
+
+### Overview
+
+This sample demonstrates a Java web application showcasing how to use spring security to validate the access token obtained using OAuth2.0 and authenticates the user.
+
+1. The Java web application obtains an:
+
+   - Id Token from Azure Active Directory (Azure AD) to sign in an user
+   - Access token that is validated using Spring security.
```

Comment:
This sample does not appear to be doing any token validation as it is only a web app signing in a user? I do not see any token validation code.

Reply:
token validation would happen in this class `WebSecurityConfigurerAdapter`, if we add more conditions in configure method (like `.access("#oauth2.hasScope('" + accessAsUserScope + "')");` in our webapi sample)

Reply:
@ramya25 Yes I understand that it would happen by adding those lines in a webapi sample. But this particular sample is not doing it so we should remove the line about AT is validated from the Readme as it is misleading to developers. Santiago also confirms below there is only logging in a user in this sample.

Reply:
sure will remove it.